pom.xml

```diff
@@ -262,6 +263,7 @@
                         <includeOnlyProperty>git.build.time</includeOnlyProperty>
                         <includeOnlyProperty>git.branch</includeOnlyProperty>
                         <includeOnlyProperty>git.dirty</includeOnlyProperty>
+                        <includeOnlyProperty>otp.graph.serialization.id</includeOnlyProperty>
```

Comment:
Is this nessessary? The resource plugin should filter all Maven properties into the resources.